lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -226,6 +226,18 @@ sub register {
         });
 
     $app->helper(
+        limit_previous_link => sub {
```

Comment:
you misunderstood me. this is a helper for the controller. I was talking about a helper for the view.